cpp/tensorrt_llm/cortex.tensorrt-llm/src/tensorrt-llm_engine.cc

```diff
@@ -47,12 +48,28 @@ namespace {
     kEndInst = 4
   };
 
+  // "<|end_of_text|>", "<|eot_id|>"
+  const std::vector<int32_t> Llama3StopWords = {128001, 128009, 1, 2};
+
   // TODO(sang) This is fragile, just a temporary solution. Maybe can use a config file or model architect, etc... 
   bool IsOpenhermes(const std::string& s) {
     if (s.find("mistral") != std::string::npos || s.find("Mistral") != std::string::npos) {
       return false;
     } 
     return true;
+  }
+    ModelType GetModelType(const std::string& s){
+    if (s.find("Meta") != std::string::npos || s.find("meta") != std::string::npos) {
```

Comment:
Should use llama3 or Llama3 for this.
We use that name in HF: https://huggingface.co/cortexso/llama3
I know this is ugly, we need to put all these things to the config file soon.